docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -46,6 +46,59 @@ determine the user credentials for the connection, often a service user. You can
 use :doc:`secrets </security/secrets>` to avoid actual values in the catalog
 properties files.
 
+.. _postgresql-sql-support:
+
+SQL support
+-----------
+
+The connector provides **read access and limited write access** to data and
+metadata in PostgreSQL. The following section lists all supported SQL statements
+and provides further details.
+
+.. important::
+
+    No other SQL statements are supported.
+
+Read access:
+
+* :doc:`/sql/select` including:
+
+  * :doc:`/sql/match-recognize`
+  * :doc:`/sql/values`
+
+* :doc:`/sql/show-catalogs`
+* :doc:`/sql/show-columns`
+* :doc:`/sql/show-schemas`
+* :doc:`/sql/show-tables`
+* :doc:`/sql/describe`
+
+Write access:
+
+* :doc:`/sql/alter-table`
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/insert`
+* :doc:`/sql/show-create-schema`
```

Comment:
Not supported.

Reply:
Sorry for the confusion, I think how GitHub shows the comment was misleading. I should've been more explicit. Only SHOW CREATE SCHEMA is not supported among these.

DROP SCHEMA, DROP TABLE (provided `allow-drop-table` is set to true) and INSERT are supported.

Reply:
Good to know. I'm going to create a separate ticket regarding the `allow-drop-table` function.

Reply:
Please also reinstate the others that are now removed (drop-schema, drop-table and insert) since they are supported.

Sorry for causing confusion.

Reply:
Done.